components/molecule/modal/src/MoleculeModal.js

```diff
@@ -45,6 +45,9 @@ const MoleculeModal = forwardRef(
     const wrapperRef = useRef()
     const ref = useMergeRefs(wrapperRef, forwardedRef)
 
+    const blockScrollPage = () => (document.body.style.overflow = 'hidden')
+    const enableScrollPage = () => (document.body.style.overflow = 'auto')
+
     const [isClientReady, setIsClientReady] = useState(false)
 
     const getContainer = () => {
```

Comment:
```js
 const getContainer = () => {
      let containerDOMEl = document.getElementById(portalContainerId)
      if (!containerDOMEl) {
        containerDOMEl = document.createElement('div')
        containerDOMEl.id = portalContainerId
        document.body.appendChild(containerDOMEl)
      }
      return containerDOMEl
    }
```
old version

it had change because b4 your last changes, the modal could be loaded in the document body or triggering and element provided.

Fix it plz